src/arena/Sprint.js

```diff
@@ -471,31 +483,41 @@ function Sprint(props) {
                     {props.redux.sprint[activeSprintId].teams[
                       index
                     ].missions.map((mission, idx) => (
-                      <Button
+                      <Chip
+                        label={`Day ${idx + 1}`}
+                        variant="options"
+                        status={displayDay === idx ? "selected" : null}
                         onClick={() => {
                           setDisplayDay(idx);
                         }}
                         // eslint-disable-next-line react/no-array-index-key
                         key={idx}
-                      >
-                        Day {idx + 1}
-                      </Button>
+                      />
                     ))}
                   </div>
                   <div className="flex">
-                    {props.user.admin === true ? (
+                    {props.user.admin === false ? (
```

Comment:
Good morning @annovo - can you change this back to only being visible for `admin`?

Reply:
Sorry, I thought I did it with “fix admin setting” commit, but apparently when I resolved the conflict between branches I missed that it changed back :(